admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -216,7 +216,7 @@ export default class OrderProductRenderer {
   }
 
   toggleProductAddNewInvoiceInfo() {
-    if ($(OrderViewPageMap.productAddInvoiceSelect).val() === 0) {
+    if (parseInt($(OrderViewPageMap.productAddInvoiceSelect).val()) === 0) {
```

Comment:
```suggestion
    if (parseInt($(OrderViewPageMap.productAddInvoiceSelect).val(), 10) === 0) {
```

Reply:
is it necessary ? I get same results with or without it :thinking: